lib/data_hygiene/document_reslugger.rb

```diff
@@ -36,7 +36,7 @@ def add_errors_if_invalid
 
       document.errors.add(:slug, "must be unique") and return if new_slug_is_a_duplicate
 
-      document.errors.add(:slug, "is invalid") unless new_slug.starts_with?("/") && new_slug !~ %r{//} && new_slug !~ %r{./\z}
+      document.errors.add(:slug, "is invalid") if new_slug.starts_with?("/") || new_slug =~ %r{//} || new_slug =~ %r{./\z}
```

Comment:
I'm not great at regex in ruby - what is this condition in words? Could we add unit tests for each of the conditions, that way at least someone like me can read the tests and understand what's invalid :-)